libs/ui-lib/lib/ocm/components/clusterWizard/ClusterDetails.tsx

```diff
@@ -72,7 +73,16 @@ const ClusterDetails = ({ cluster, infraEnv }: ClusterDetailsProps) => {
     async (params: ClusterCreateParamsWithStaticNetworking, addCustomManifests: boolean) => {
       clearAlerts();
       try {
-        const cluster = await ClustersService.create(params);
+        const searchParams = new URLSearchParams(location.search);
+        const isAssistedMigration = searchParams.get('source') === 'assisted_migration';
+        let createParams = params;
+        if (isAssistedMigration) {
+          createParams = {
+            ...params,
+            tags: 'assisted_migration',
```

Comment:
I would like to see this as a constant, not a string.

And it would be nice to handle this inside of the `ClusterDetailsService.getClusterCreateParams()` function that is called from `ClusterDetailsForm.tsx` when the form is submitted. It would be better to handle everything related to cluster creation parameters in one place instead of multiple. Pass the `isAssistedMigration` value as a parameter of this function perhaps.

Also, this replaces the `AI_UI_TAG` that is added in OCM. That is correct I assume?